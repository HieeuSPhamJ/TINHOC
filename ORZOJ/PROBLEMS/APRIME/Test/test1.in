281
32405 8549 18672 14031 16992 22548 1520 9959 7698 23888 22661 1671 32335 12155 14491 21694 29055 16560 14322 25628 10366 17574 10353 14265 28565 5591 31934 12664 13226 30177 5899 30093 26547 11564 31283 10045 24177 19917 4357 25317 27738 7594 32735 6462 26071 27615 13440 12255 19887 21857 10263 2820 18371 15585 1951 25741 9213 31220 13935 5194 5507 880 24333 11637 27184 6425 1604 5504 22084 12413 21622 30738 10330 10524 2355 15436 31131 3020 19721 24400 11826 22876 21918 22532 23039 11384 4093 26643 23952 13004 17710 32309 20301 16749 27913 3505 1266 2164 9035 32544 20803 10732 29418 7795 18939 14318 27558 10807 10757 24074 13793 26648 9198 14825 6320 25252 14491 10033 9778 13394 26167 681 545 963 17264 10398 30374 5447 23267 12933 19314 5539 23252 14305 4874 530 27939 5735 25186 20873 19323 1376 9902 17215 25549 30800 3445 30317 8445 5333 13031 12114 11227 16337 2672 28027 14420 1483 13709 14145 13034 25235 15583 16019 25185 11301 24446 17142 20923 9424 27791 21705 5735 30872 23321 24014 18102 10712 5140 4838 13454 12588 8899 13084 16482 21645 16196 14309 22625 31166 26470 23206 30702 30696 31162 4983 15185 11393 1773 29991 23468 22385 19796 2688 10032 632 4809 6348 18832 30666 20662 27544 30902 3069 26223 3511 22966 16165 17861 5002 12800 31038 19928 23167 14553 26058 18476 25048 7436 12986 15359 8366 4828 13970 3034 32053 32318 16841 4455 18753 3397 8046 11138 26633 22238 2322 29330 299 20925 4577 1818 25841 14231 19730 11915 1491 10386 13186 12893 11069 8954 3227 13661 16786 9013 24187 29513 27504 32761 10578 7834 14390 22204 29196 20641 9155 7276 24625 16341 17281 9825 