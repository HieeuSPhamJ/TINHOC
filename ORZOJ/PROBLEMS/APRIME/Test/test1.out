70
34
0
31
0
2
5
37
2
12
45
12
91
33
55
12
62
2
1
10
12
2
25
39
61
27
14
7
10
44
34
59
40
4
64
46
52
45
20
53
1
11
52
1
67
54
3
54
34
43
19
5
28
38
15
75
18
23
40
3
30
5
35
29
12
40
5
6
15
40
13
3
9
1
19
15
57
3
46
12
0
17
1
7
50
6
31
29
1
3
14
75
29
53
84
28
2
7
40
1
73
8
2
50
55
10
4
40
35
11
40
14
2
51
12
6
55
50
17
6
60
3
9
7
7
0
13
21
63
37
1
26
6
59
4
5
56
30
18
54
25
3
5
70
85
15
17
77
32
25
47
1
50
50
4
96
19
14
36
52
12
58
51
40
53
24
13
0
71
8
55
57
30
9
44
9
2
6
8
4
11
1
39
10
3
39
10
38
45
13
12
21
2
1
12
17
57
44
7
41
22
70
21
4
1
5
19
1
9
2
7
14
11
13
36
23
17
75
57
4
15
1
16
74
28
2
13
13
8
13
31
8
5
11
5
90
19
48
17
33
16
1
15
54
21
0
30
8
47
20
3
60
32
18
38
9
2
12
53
34
5
16
44
14
46
66
67
3
89
1
7
5
9
1
83
40
8
96
30
59
42
